.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ jobs:
         environment:
           TARGET_ARCH: x64
           COMPONENT: shared_library
+          CHROMIUM_BUILDTOOLS_PATH: /home/builduser/project/src/buildtools
```

Comment:
It's a bummer that we need to set this everywhere; could we perhaps convince gclient to find the root another way? Looking through [gclient_utils.py](https://chromium.googlesource.com/chromium/tools/depot_tools/+/6f64a0522bb8b6d20686028768024ebce5cfbba9/gclient_utils.py#622), it looks like it looks for a `.gclient` file and associated `.gclient_entries` (created by `gclient sync`). Is it possible the tarball is missing those files?

Reply:
@nornagon I don't really think its a big deal to set the environment variable.

I did look for a .gclient file and a .gclient_entries and those files do not exist at the root of the src directory after running gclient sync.  Maybe it has to do with how we call gclient sync?

Reply:
They're siblings to `src/`:

```
.
├── .gclient
├── .gclient_entries
└── src
```